doc/configuration.txt

```diff
@@ -2958,6 +2959,10 @@ maxzlibmem <number>
   with "show info" on the line "MaxZlibMemUsage", the memory used by zlib is
   "ZlibMemUsage" in bytes.
 
+mptcp
+  Enables the use of the Multipath TCP (MPTCP) protocol. This option is only
+  available when support for MPTCP was built in.
```

Comment:
what do you mean with "MPTCP was built in"?

if the kernel supports it? Currently limited to Linux?